frontend/src/k8sTypes.ts

```diff
@@ -378,11 +378,13 @@ export type InferenceServiceKind = K8sResourceCommon & {
   };
   spec: {
     predictor: {
+      tolerations?: PodToleration[];
       model: {
```

Comment:
wasn't this the _type_ ? not actually the text `model`?

I don't have the doc link anymore ... but I thought this was something like `triton` or something like that... I may be mis-remembering, so ignore me if I am.

Reply:
That's a great question, we've had the model placeholder for modelmesh and there are examples with model, but ill defer to @Xaenalt and @Jooho that question.

Reply:
@andrewballantyne I think you saw this kind of example
~~~
apiVersion: "serving.kserve.io/v1beta1"
kind: "InferenceService"
metadata:
  name: "flower-sample"
spec:
  predictor:
    tensorflow:
      storageUri: "gs://kfserving-examples/models/tensorflow/flowers"
~~~
It is an old Schema. The new schema looks like this:
~~~
apiVersion: "serving.kserve.io/v1beta1"
kind: "InferenceService"
metadata:
  name: "flower-sample"
spec:
  predictor:
    model:
      modelFormat:
        name: tensorflow
      storageUri: "gs://kfserving-examples/models/tensorflow/flowers"
~~~

Doc:https://kserve.github.io/website/0.10/modelserving/v1beta1/tensorflow/